identity_vc/src/common/context.rs

```diff
@@ -0,0 +1,36 @@
+use identity_core::common::Object;
+use std::fmt;
+
+use crate::common::URI;
+
```

Comment:
The DID standard doesn't mention object-based contexts but the implementation **could** be flexible. The DID document could use *this* `Context` (`context: OneOrMany<Context>`) with a few minor changes.

It may not be entirely useful without JSON-LD support **but** avoids ser/deserialization errors on spec-compliant data.

@tensor-programming thoughts?

Reply:
@JelleMillenaar I put this particular change in a [new branch](https://github.com/iotaledger/identity.rs/tree/feat/shared-context). It's a bit more involving with `serde_diff` and depends on diff support for `Value` which @tensor-programming is currently working on(?)

Reply:
Yeah sorry, didn't see this conversation until just now.  Yeah I am working on the diff implementation and I will do something about making serde's `Value` a priority.  The context as it is looks good right now and we can always add more flexibility later.  Ill do a full review in the morning.